packages/office-addin-test-server/src/testServer.ts

```diff
@@ -3,37 +3,40 @@
 
 import * as cors from "cors";
 import * as express from "express";
+import * as http from "http";
 import * as https from "https";
 import * as devCerts from "office-addin-dev-certs";
 
-export const defaultPort: number = 4201;
+export const defaultHttpPort: number = 4200;
+export const defaultHttpsPort: number = 4201;
 
 export class TestServer {
     private jsonData: any;
-    private port: number;
+    private httpPort: number;
+    private httpsPort: number;
     private testServerStarted: boolean;
     private app: express.Express;
     private resultsPromise: Promise<JSON>;
-    private server: https.Server;
+    private httpServer: http.Server;
+    private httpsServer: https.Server;
 
     constructor(port: number) {
         this.app = express();
         this.jsonData = {};
-        this.port = port;
+        this.httpsPort = port;
+        this.httpPort = defaultHttpPort;
         this.resultsPromise = undefined;
         this.testServerStarted = false;
     }
 
-    public async startTestServer(mochaTest: boolean = false): Promise<boolean> {
+    public async startTestServer(mochaTest = false /* deprecated */): Promise<boolean> {
```

Comment:
Use JSDoc comments to annotate this.
/** 
  * @param mochaTest @deprecated
 * 
*/

It would be nice to have all exported functions have JSDoc comments so you get intellisense.

Reply:
You'll have to check if that syntax actually works. I think it does but I could be wrong.

Reply:
Good idea - I will add JSDoc comments